client/src/context/UserContext.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, createContext } from 'react';
+import PropTypes from 'prop-types';
+
+export const UserContext = createContext();
+
+export function UserContextProvider(props) {
+  const [url, setUrl] = useState('');
```

Comment:
Might consider renaming this var to sometime like imgUrl/setImgUrl after we merge because there are some vars named url that deal with the users' unique urls. Depends on if they'll end up colliding on any pages though I suppose.

Reply:
Sure that's an easy fix.